### PR TITLE
add advanced escrow to CI

### PR DESCRIPTION
**Pull request by shunjizhan**
## Change
added advanced escrow to CI

## Test
CI tests passed https://github.com/AcalaNetwork/bodhi.js/runs/7070558298?check_suite_focus=true

### CODE REVIEW

**Comment by ThunderDeliverer** [comment]
Let's wait for `MandalaAddress` to be released and merge it then 👌

File: advanced-escrow/test/loop.js [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-  let count = 0;
-
-  while (true) {
-    await sleep(interval);
-    await api.rpc.engine.createBlock(true /* create empty */, true /* finalize it*/);
-    console.log(`Current number of force generated blocks: ${++count}`);
-  }
-};
-
-loop();
```
Comment (ThunderDeliverer) [inline advanced-escrow/test/loop.js:22]: @shunjizhan why did you remove the `loop` utility? It is still needed for the user journey to be completed

**Comment by shunjizhan** [comment]
ooooops I didn't realize it is still needed for this specific example, thought it was legacy file :sweat_smile:
